qemu/tests/cfg/slof_device_tree.cfg

```diff
@@ -0,0 +1,8 @@
+- slof_device_tree: install setup image_copy unattended_install.cdrom
```

Comment:
Power only support linux guest now, I think we can add the filter to shared/cfg.